src/components/network/network-table.js

```diff
@@ -1104,6 +1115,18 @@ const NetworkTable = (props) => {
                                 child={checkListColumnsNames()}
                             />
                         </Grid>
+                        {!allowModificationsOnNode(
+                            props.workingNode,
+                            props.selectedNode
+                        ) &&
+                            props.selectedNode?.type !== 'ROOT' && (
```

Comment:
If someday a node is readOnly, but its values are still correct, then this would not make sense : as is, we are testing if we have the rights to modify the node, but somehow the message shows the node is invalid.

Maybe we should create a new function "isNodeValid" or something, and then inside this function do these tests. This way, the message "invalid node" is coherent with the test we do.

Reply:
Done